project_event/models/project_task.py

```diff
@@ -280,7 +285,7 @@ def write_activity(self, vals):
         return super(Task, self).write(vals)
 
     @api.multi
-    def write_task(self, vals):
+    def write_main_task(self, vals):
```

Comment:
@eilst: method name write_main_task () look good but main_task is not use  anywhere so you can write as 
return self.get_main_task().main_task.write(vals)

Reply:
True, you created that method, and I thought the same about just used once. But actually, since main task is managed in a different manner than a normal task, I thought its likely that we are going to refer this method later. So in my opinion it should stay, It makes the code more clear.